cmake/OpenCVModule.cmake

```diff
@@ -573,6 +575,10 @@ macro(ocv_create_module)
     "${OPENCV_CONFIG_FILE_INCLUDE_DIR}/cvconfig.h" "${OPENCV_CONFIG_FILE_INCLUDE_DIR}/opencv2/opencv_modules.hpp"
     ${${the_module}_pch})
 
+  set(OPENCV_MODULE_${the_module}_INTERFACE_DEFS ${OPENCV_MODULE_${the_module}_INTERFACE_DEFS}
+    CACHE INTERNAL "Public compile definitions of ${full_modname} module")
+  target_compile_definitions("${the_module}" PUBLIC ${OPENCV_MODULE_${the_module}_INTERFACE_DEFS})
```

Comment:
`target_compile_definitions` is only available since CMake 2.8.11, while the declared minimum is 2.6.3.